requirements.txt

```diff
@@ -4,14 +4,14 @@ azure-common==1.1.8
 azure-nspkg==2.0.0
 azure-storage==0.36.0
 boto3==1.6.0
-botocore==1.10.8
+botocore==1.9.23
```

Comment:
Does `botocore 1.9.23` have support AWS Secrets Manager?

This may cause deploy to fail.

Reply:
#1192 

It turns out pip is pretty bad at dependency management.